profiles/seccomp/default_linux.go

```diff
@@ -42,6 +40,9 @@ func arches() []types.Architecture {
 
 // DefaultProfile defines the allowed syscalls for the default seccomp profile.
 func DefaultProfile() *types.Seccomp {
+	if !isEnabled() {
```

Comment:
wondering; what would happen if we didn't even check, and just passed this down to containerd/runc (having them deal with wether or not to do something with it)?

Reply:
Looks like it would error out: https://github.com/opencontainers/runc/blob/2c632d1a2de0192c3f18a2542ccb6f30a8719b1f/libcontainer/seccomp/seccomp_unsupported.go#L16

Reply:
Sorry that's different... when runc is not compiled with seccomp support.

Reply:
Most likely what will happen here is the kernel will throw an error.

Reply:
That's if runc applies it; I wonder if runc should do the check, and skip that part of the spec if it's not supported 🤔

Basically; if could have multiple OCI runtimes configured, and the docker daemon doesn't know what they support (hope you follow my train of thought)